app/views/assignments/_form.html.erb

```diff
@@ -18,40 +19,73 @@
+  <%= f.select :file_structure, Assignment.options_for_file_structure %>
+  </div>
+  <div>
+    <%= f.label :file, "Student submissions (zip) \u24D8", title: "The zip file should contain files according to the way you selected the \"File Structure \" option above. \n\nIf you selected \"By Files\", then, all the files in the zip folder will be compared with each other. Name your file as \"skeleton \" for the codes you wish to exclude from the system  \n\nIf you selected the other option (\"By Directory\"), ensure that the zip file contains one folder for each student's code files. Ensure that the folders are named according to student ID and name your folder as \"skeleton\" for the codes you wish to exclude from the system" %>
     <%= f.file_field :file %>
   </div>
+
   
   <div>
-  <%= f.label :mapbox, "Upload map file?" %>
+  <%= f.label :mapbox, "Add more options? \u24D8", title: "Check the box to include additional optional options"%>
   <%= f.check_box :mapbox,{}, "Yes", "No"%>
   </div>
   <div id="map_file_details" style="display:none;">
-  <%= f.label :mapfile, "Mapping file (csv) [?]", title: "The zip file should contain one folder for each student's code files; folders are named according to student ID; Name your folder as \"skeleton\" for the codes you wish to exclude from the system" %>
+  <%= f.label :mapfile, "Mapping file (csv) \u24D8", title: "If you have a map file that maps the filenames to other attributes such as student's names, github handle etc, upload the file here. Ensure that the file is a csv file and follows the conventions according to the user guide." %>
   <%= f.file_field :mapfile, accept: '.csv' %>
-
+  </div>
+  <div id="file_to_ignore_details" style="display:none;">
+  <%= f.label :filesIgnore, "Files to be ignored \u24D8" , title: "If you want to ignore certain files from the code zip file that you uploaded, add those filenmes under here. \n\nIf you want to ignore filenames that exactly match a certain filename, enter the filename as it is. For instance, entering \"question1\", will mean that all the filenames that are named as \"question1\" (irrespective of their file extensions) will all be ignored. \n\nIf you want to ignore filenames that partially contain a substring, add \"/ \" in front of the filename. For instance, if you enter, \"/question1\", filenames such as \"alice_question1.py\", \"bob_question1Test.c\", \"question1_charlie.java\" will all get ignored. \n\nIf you need to add muliple regex, leave a space in between them. For instance, if you enter \"question1 /question2 \", filesnames that either exactly match \"question1 \" or contain the substring \"question2 \" will all get ignored"  %>
+  <%= f.text_field :filesIgnore %>
+  </div>
+  <div id="file_to_process_details" style="display:none;">
```

Comment:
May need to be clear which of `ignore` or `to process` has precedence.  Should be ordered as such too (overriding one listed first).